repository/src/main/java/org/alfresco/repo/content/ContentServiceImpl.java

```diff
@@ -591,14 +591,24 @@ public boolean isContentDirectUrlEnabled()
      * {@inheritDoc}
      */
     @Override
+    @Deprecated
     public boolean isContentDirectUrlEnabled(NodeRef nodeRef)
+    {
+        return isContentDirectUrlEnabled(nodeRef, ContentModel.PROP_CONTENT);
+    }
```

Comment:
This should be moved to ContentService.java as a `default` method call.
See `ContentService.requestContentDirectUrl(NodeRef nodeRef, boolean attachment)` for an example.

Reply:
I wouldn't change interface method in a public project and hardcode that here. 
Very unlikely, but there may be custom implementations for this method which then would be broken. 
Same bit of code resides in `ContentServiceImpl` for this method, which, if you ask me, is safer.

Reply:
This should be resolved now but please double check it is what you meant